MaaSSim/traveller.py

```diff
@@ -185,6 +185,7 @@ def pax_action(self):
         """main routine of the passenger process,
         passes through the travellerEvent sequence in time and space"""
         self.update(event=travellerEvent.STARTS_DAY)
+        self.sim.requests.loc[len(self.sim.requests.index) + 1] = self.request  # append request
```

Comment:
Why do we need it here? Won't it screw other routines?
Maybe it is enough to put it in line 192 (after else)

Reply:
plus, definitely, you shall add request only for 'schedule_leader', so put this behind: 
"if self.schedule_leader:"

Reply:
I don't exactly recall why I moved it, I think it relates to the trav_out function, but indeed it is not needed there